### ISSUE TITLE
MA and (Conditional) Emergency Stop Message 

### ISSUE THREAD

**Issue by JakobGartner** [openETCS/modeling#531]
In the SRS, 3.10.2.4 states: 
New movement authority received after any accepted emergency stop message and before the emergency message has been revoked, shall be rejected.

In the JRU data, the train accepts MA after having accepted message 15 and having acknowledged it.
The Conditional Emergency Stop has been received, accepted and has never been revoked.

I assume that a Conditional  Emergency Stop becomes irrelevant when we pass its location?
How long do I have to store it after having passed the location? I don't find any info on the topic.

I will assume that I have to reject MA from track when a Conditional Emergency Stop has been received and until the position has been passed. 

Right? .... Wrong?

**Comment by MarcBehrens** [comment]
Yes, as long as the train's *min safe front end* (3.10.2.2) did not pass the announced position for the conditional emergency stop the train accepts the emergency stop.

This could mean e.g. a red signal ahead.

Following 3.10.2.4 a new MA is not accepted, as long a e.g. the signal does not turn green.

**Comment by JakobGartner** [comment]
I have understood so far.

The problem is that on the track, we have several Message 15 (Conditional Emergency Stop) that are received and acknowledged)by the train.
Unfortunately, no details are visible in the JRU data for the variables set in message 147 by the train.

Later on the track, the train accepts (=acknowledges) Message 3 (MA).
The conditional emergency stop has never been revoked.

The train seems not to have stopped either... 

I have to analyse further, but it seems that the train *never* stops based on emergency stop message, which means that it probably has always already passed the announced position with its safe front end...
